ouroboros-consensus-shelley/src/Ouroboros/Consensus/Shelley/HFEras.hs

```diff
@@ -65,5 +65,12 @@ instance
   (TPraos.PraosCrypto c, Signable (DSIGN c) (Hash (HASH c) EraIndependentTxBody)) =>
   ShelleyCompatible (TPraos c) (AlonzoEra c)
 
+-- This instance is required since the ledger view forecast function for
```

Comment:
a TODO to follow-up on (Jira ticket?)

Reply:
Currently: we create the `Praos` ledger view by calling `cardano-ledger` code that creates a `TPraos` ledger view and then projecting out the salient subset. Hence `Babbage` needs this instance despite using `Praos` not `TPraos`.

The Jira ticket for deciding how to extract the `Praos`-parts of that `cardano-ledger` code should include removing this instance as a small finishing touch.

Reply:
I created https://input-output.atlassian.net/browse/CAD-4227 for this

Reply:
This issue is now tracked in https://github.com/input-output-hk/ouroboros-network/issues/3878